source/blog/2018-07-19-ovn-dpdk.html.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Upgraded DPDK support in oVirt
+author: lgoldber
+tags: OVS-DPDK, Open vSwitch, OVN, ansible
+date: 2018-07-19 10:00:00 CET
+comments: true
+published: true
+---
+[DPDK](http://dpdk.org/) (Data Plane Development Kit) provides high-performance packet processing libraries and user space drivers.
+
+[Support for DPDK](https://www.ovirt.org/blog/2017/09/ovs-dpdk/) in oVirt was introduced in 2017, and is now upgraded in terms of both deployment via Ansible and usage via [Open Virtual Network](http://www.ovn.org/).
+
+While still experimental, OVN-DPDK in oVirt is now available in both the stable and master oVirt branches. 
```

Comment:
Since this is for end users, change "stable" to what exact version the user can use.

Reply:
Done